Assets/Scripts/Player/Controls.cs

```diff
@@ -0,0 +1,307 @@
+// GENERATED AUTOMATICALLY FROM 'Assets/Scripts/Player/Controls.inputactions'
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.Utilities;
+
+public class @Controls : IInputActionCollection, IDisposable
+{
+    public InputActionAsset asset { get; }
+    public @Controls()
+    {
+        asset = InputActionAsset.FromJson(@"{
```

Comment:
How likely is this to change? It might be better to stick this in a JSON file and load it in separately if it will change.

Reply:
Controls.cs is recreated every time the Control (Input Actions file from the input system) changes. Not sure if it'll work in a JSON file

Reply:
Alright we'll merge in and I'll add a TODO for looking into it 👍